tools/config-markdown.py

```
@@ -1,207 +1,169 @@
+            md.append(f"<!--- footer STOP from {rel_path} --->\n")
+
     if 'sub' in item:
+        grouped_subs = {}
         for sub_item in item['sub']:
-            user_content += create_markdown_user(sub_item, level + 1)
-    
-    return user_content
+            prefix = sub_item['id'][:3].upper()
+            grouped_subs.setdefault(prefix, []).append(sub_item)
+
+        for prefix, sub_items in grouped_subs.items():
```

Ruff (0.8.2)

Loop control variable `prefix` not used within loop body

(B007)